### PR TITLE
Convert README to markdown

### PR DESCRIPTION
**Pull request by gavlyukovskiy**
Used https://github.com/opendevise/downdoc to get rid of that abomination

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,46 +111,45 @@ Example showing an allow-list based approach of masking JSON where additionally
...
+```json
 {
     "customerId": "123 789 456",
     "customerDetails": {
-        "firstName": "*****",
-        "lastName": "**************",
-        "email": "***************************",
+        "firstName": "*******",
+        "lastName": "**********************",
+        "email": "*********************************************",
```
Comment (Breus) [inline README.md:130]: @gavlyukovskiy 🤣

File: README.md [hunk 2]
```diff
@@ -14,28 +14,27 @@ The library provides a convenient API and its implementation is focused on maxim
...
-- **Block-list** (`masked`) or **allow-list** (`allowed`) interpretation of target key set (default: `masked`)
-- Masking valid JSON will always result in valid JSON
-- The implementation only supports JSON in UTF-8 character encoding
+* Mask ***strings*** in any JSON structure that correspond to a configured target key (default)
+* Mask all values in ***(nested) JSON arrays*** that correspond to a configured target key (default)
+* Maks all values in ***(nested) JSON objects*** that correspond to a configured target key (default)
+* Mask ***numbers*** in any JSON structure that correspond to a configured target key (optional)
+* ***Obfuscate*** the original length of the masked value by using a fixed-length mask (optional)
+* Target key ***case sensitivity configuration*** (default: `false`)
+* ***Block-list*** (`masked`) or ***allow-list*** (`allowed`) interpretation of target key set (default: `masked`)
```
Comment (Breus) [inline README.md:25]: All these bold words are wrong now?

Comment (gavlyukovskiy) [reply README.md]: Looks like they are ***bold cursive*** now 😄 I've changed it to just **bold**. Also found that image links we a bit wrong, fixed those too.

**Review by Breus — APPROVED** [review]

**Comment by gavlyukovskiy** [comment]
Did couple more updates for minor stuff, looks almost the same now:
https://github.com/Breus/json-masker/blob/convert-to-md/README.md
https://github.com/Breus/json-masker/blob/master/README.adoc

**Pull request merged**